src/tools/base/BaseBrushTool.js

```diff
@@ -141,7 +141,13 @@ class BaseBrushTool extends BaseTool {
    */
   // eslint-disable-next-line no-unused-vars
   passiveCallback(evt) {
-    external.cornerstone.updateImage(this.element);
+    try {
+      external.cornerstone.updateImage(this.element);
```

Comment:
I really wish `updateImage` and similar methods were safe to call with elements that don't have a loaded image or are not called with an enabled element.

Reply:
Totally agree.

Particularly `updateImage`. It doesn't access the image itself, it just calls `drawImage` which also doesn't touch the image but sets `enabledElement.needsRedraw`.

`needsRedraw` is then checked in the run-loop which also checks to make sure the image is loaded.

https://github.com/cornerstonejs/cornerstone/blob/f3b4accef3a700b0719e8373c08c414a1448b563/src/enable.js#L97-L99

There's nothing here that would break if `updateImage` was called when no image is loaded.